### PR TITLE
EASY-2390: SMD: Zet dans-doi-action op 'update' als DOI en BASE-REVISION zijn gegeven

### PR DESCRIPTION
**Pull request by vesaakerman**
fixes EASY-2390

#### When applied it will
* set `dans-doi-action` to '`update`' when the deposit has a `base-revision` and a `DANS Doi.`
 

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-dtap                      | [PR#480](https://github.com/DANS-KNAW/easy-dtap/pull/480)

### CODE REVIEW

File: src/main/assembly/dist/cfg/application.properties [hunk 1]
```diff
@@ -5,3 +5,4 @@ auth.ldap.url=ldap://localhost
 auth.ldap.user=cn=ldapadmin,dc=dans,dc=knaw,dc=nl
 auth.ldap.password=changeme
 audio-video.ffprobe=/usr/bin/ffprobe
+dans.doi=10.17026/
```
Comment (lindareijnhoudt) [inline src/main/assembly/dist/cfg/application.properties]:
```suggestion
dans.doi.prefix=10.17026/
```
this should be propagated throughout

Comment (lindareijnhoudt) [inline src/main/assembly/dist/cfg/application.properties]: Remember to add extra application properties also to dtap

**Review by lindareijnhoudt — CHANGES_REQUESTED** [review]

File: src/main/assembly/dist/cfg/application.properties (hunk 1 above)
Comment (vesaakerman) [reply src/main/assembly/dist/cfg/application.properties]: done

Comment (vesaakerman) [reply src/main/assembly/dist/cfg/application.properties]: also done

**Pull request merged**